### PR TITLE
Slash response

### PR DESCRIPTION
**Pull request by Isez98**
Integrated the legacy code to the moduled project. 

Basic functionality of the slash-event response was integrated and the _loop_ file was deleted for now being obsolete. 

The _legacy-code_ folder was kept due to possibly having reusable code for the template of the slack message.

### CODE REVIEW

File: devsearchbot.go [hunk 1]
```diff
@@ -1,9 +1,10 @@
 package devsearchbot
 
 import (
-
-  "github.com/slack-go/slack"
-  "github.com/elvisgastelum/utils"
+	"log"
+	"net/http"
+//	"github.com/slack-go/slack"
+//  "github.com/elvisgastelum/utils"
```
Comment (elvisgastelum) [inline devsearchbot.go]: No las comentes, eliminalas si no las ocupamos, En caso de ocuparlas en un futuro se escriben de nuevo

File: devsearchbot.go [hunk 2]
```diff
@@ -19,12 +20,29 @@ type Bot struct{}
...
-  go rtm.ManageConnection()
-
-  loop(rtm)
-}
-
-
+	// token := utils.Getenv("SLACK_ACCESS_TOKEN")
+  // api := slack.New(token)
+  // rtm := api.NewRTM()
+	// go rtm.ManageConnection()
```
Comment (elvisgastelum) [inline devsearchbot.go]: Va, hagamos esto, todo mi codigo antiguo, es como si no existiera, borralo por completo de ser necesario

File: handle-message.go [hunk 3]
```diff
@@ -2,31 +2,25 @@ package devsearchbot
...
 import (
 	"log"
-	"regexp"
-	"strings"
+	"net/http"
+	"bytes"
+	"fmt"
 
-	"github.com/slack-go/slack"
+//	"github.com/slack-go/slack"
```
Comment (elvisgastelum) [inline handle-message.go]: Lo mismo aqui

**Review by elvisgastelum — CHANGES_REQUESTED** [review]
Cambios menores

**Comment by Isez98** [comment]
Resolved request changes

**Review by elvisgastelum — APPROVED** [review]

**Pull request merged**